include/ck/tensor_operation/gpu/grid/gridwise_batched_gemm_softmax_gemm_xdl_cshuffle_v1.hpp

```diff
@@ -720,6 +720,8 @@ struct GridwiseBatchedGemmSoftmaxGemm_Xdl_CShuffle
             static_for<0, acc_thread_buf.Size(), 1>{}(
                 [&](auto i) { acc_element_op(acc_thread_buf(i), acc_thread_buf[i]); });
 
+            block_sync_lds(); // wait for lds read in gemm0 blockwise gemm
```

Comment:
So this sync is needed, regardless of whether verification passes or not, right?

Reply:
Yes. My last refactor overlaps softmax workspace with AB block. Although the issue at hand is still there, this is still needed.